app/models/issue.rb

```diff
@@ -48,4 +48,32 @@ def transition_callback
     puts "Current event: #{aasm.current_event}"
     puts "Transition Callback: from #{aasm.from_state} to #{aasm.to_state}"
   end
+
+  def inform_status_change
+    watchers.includes(:user).each do |watcher|
+      # IssueMailer.delay.status_change(watcher.user, self, company.subdomain)
+      IssueMailer.with(
+        user: watcher.user,
```

Comment:
do not pass whole resource to the mailer. There is the possibility that user is deleted before even running delayed jobs. Do this same for issue or any other resources.

Reply:
ok